### ISSUE TITLE
Save/load current state

### ISSUE THREAD

**Issue by ahitrin** [ahitrin/clokado#1]
Currently, no save/load function is provided. The only working way is:
1. Show tree from REPL and copy-paste it to somewhere
2. Start new REPL session and copy-paste saved tree into it

**Comment by ahitrin** [comment]
Possible file layout: CSV-like

```
1,"Mikado goal",true,
2,"Blocking goal",true,"1"
3,"Closed blocking goal",false,"1,2"
```

**Comment by ahitrin** [comment]
Next idea: apply save/load no the fly:
- when starting new tree with `mikado!` fn, just open existing file with structure or create the new one
- after any action (`add!`, `link!`, `unlink!`, etc.) save current version into the file, and then redraw

**Comment by ahitrin** [comment]
Implemented in `0.3.5`
